miprometheus/problems/video_to_class/seq_mnist_to_class/permuted_sequential_row_mnist.py

```diff
@@ -60,9 +60,9 @@ def __init__(self, params):
 
                 - ``self.data_definitions`` :
 
-                    >>> self.data_definitions = {'images': {'size': [-1, 3, -1, -1], 'type': [torch.Tensor]},
-                    >>>                          'mask': {'size': [-1, -1, -1, -1], 'type': [torch.Tensor]},
-                    >>>                          'targets': {'size': [-1, 1], 'type': [torch.Tensor]},
+                    >>> self.data_definitions = {'images': {'size': [-1, 28, 1, 1, 28], 'type': [torch.Tensor]},
+                    >>>                          'mask': {'size': [-1, 28, 1], 'type': [torch.Tensor]},
```

Comment:
So in here the dimensions are BATCH (-1) by SEQ_LEN (28) by OUTPUT DIM (1)

Good! :)